content/en/blog/2022/enhancements-process-update.md

```diff
@@ -0,0 +1,19 @@
+# Release Opt-in Process Update
+
+## Context and Motivations
+Since the inception of the Kubernetes release team, we have used a spreadsheet to keep track of enhancements for the release. The project has scaled massively in the past few years, with almost a hundred enhancements collected for the 1.24 release. This process has become error-prone and time consuming. A lot of manual work is required from the release team and the SIG leads to populate KEPs data in the sheet. We have received continuous feedback from our contributors to streamline the process.
+
+Starting in the 1.26 release, we are replacing the sheet with an automated Github project.
```

Comment:
For clarity, I would suggest `Starting at the beginning of the 1.26 release ....` or `Starting with the 1.26 release ....`